compliance/rio/src/test/java/org/eclipse/rdf4j/rio/helpers/RDFParserHelperTest.java

```diff
@@ -108,7 +108,7 @@ public final void testCreateLiteralLabelOnly()
 				valueFactory);
 
 		assertEquals(LABEL_TESTA, literal.getLabel());
-		assertFalse(literal.getLanguage().isPresent());
+		assertFalse(literal.getLanguage() != null);
```

Comment:
Just backporting will mean the asserts don't have to be read by humans as often so sounds good to me.